README.md

```diff
@@ -5,6 +5,68 @@ https://flutter.dev/
 Flutter in an open Source SDK for creating high-performance mobile apps for IOS and Android.
 The Flutter makes it easier for you to build user interfaces, while reducing the amount of code required to create and update your app.
 
+## Why use Flutter?
+
+- Flutter can be used to build applications for Android and IOS
+- The Dart language used in Flutter is quite similar to Java
```

Comment:
```suggestion
- The Dart programming language used in Flutter is object oriented and familiar to most developers.
```
Comparing `Dart` to `Java` might be _accurate_ but it's not a _favourable_ comparison. 
A lot of people who write code for a living *hate* `Java` because it's _very_ repetitive and has lots of boilerplate. https://www.reddit.com/r/programming/comments/9dzpu/ask_reddit_why_does_everyone_hate_java
The people who "love" `Java` is because they have https://en.wikipedia.org/wiki/Stockholm_syndrome

Reply:
Even in the Flutter Architecture video and compared to C++ or Java as being object oriented.